RSSReader-Maruyama/DataManager/UserDataManager.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  LoginChecker.swift
+//  RSSReader-Maruyama
+//
+//  Created by 丸山翔 on 2020/09/03.
+//  Copyright © 2020 丸山翔. All rights reserved.
+//
+
+import UIKit
+
+class UserDataManager: NSObject {
+    
+    private let UD_USER_DATA_KEY = "userdata"
```

Comment:
```suggestion
    private let UD_USER_DATA_KEY = "userData"
```
lowerCamelにしましょう

Reply:
```suggestion
    private let USER_DATA_KEY = "userData"
```
接頭辞UD_はどういった意味で付けているのでしょうか？
UserDataの略称でしょうか？必要ないように思えますが

Reply:
lowerCamelで、かしこまりました。
UDは「UserDefaults」の略で付けました。削除したほうがよろしいでしょうか？

Reply:
UserDefaultsのKeyで有ることを明示したいのであれば下記のようにExtensionを実装することをオススメします。
```swift
extension UserDefaults {
    enum Keys : String{
        case userData
    }
}

// example
let userData = UserDefaults.standard.data(forKey: UserDefaults.Keys.userData.rawValue)
```

Reply:
確かにこちらの方が、より理解りやすく便利ですね。
ありがとうございます。